docs/topics/ksp/ksp-quickstart.md

```diff
@@ -1,88 +1,181 @@
 [//]: # (title: KSP quickstart)
 
-Get a [sample processor](https://github.com/google/ksp/releases/download/1.5.30-1.0.0-beta09/playground.zip) to check out.
+Get a [sample processor](https://github.com/google/ksp/releases/download/%kspSupportedKotlinVersion%-%kspVersion%/playground.zip) to check out.
```

Comment:
Maybe some more info?
If I understand correctly, the article means the following: you can start by creating a new processor. Or you can look through the sample ...

Reply:
> For a quick start, you can create your own processor or get a [sample one](...).

[Fixed.](https://github.com/JetBrains/kotlin-web-site/pull/2570/commits/290ab2a7217ebfb471b01f40590d932479301f1e)